packages/core/src/browser/tree/tree-model.ts

```diff
@@ -155,14 +155,7 @@ export class TreeModelImpl implements TreeModel, SelectionProvider<ReadonlyArray
         this.toDispose.push(this.selectionService);
 
         this.toDispose.push(this.expansionService);
-        this.toDispose.push(this.expansionService.onExpansionChanged(node => {
-            this.fireChanged();
-            if (!node.expanded && [...this.selectedNodes].some(selectedNode => CompositeTreeNode.isAncestor(node, selectedNode))) {
```

Comment:
only extract lines from 160-164, lisnener should be installed always in the super class and `fireChanged` should be fired always